internal/handlers/tigris/msg_update.go

```diff
@@ -210,7 +211,15 @@ func (h *Handler) update(ctx context.Context, sp fetchParam, doc *types.Document
 	h.L.Sugar().Debugf("Update: %s", u)
 
 	res, err := h.driver.UseDatabase(sp.db).Update(ctx, sp.collection, f, u)
-	if err != nil {
+	switch err := err.(type) {
+	case nil:
+		// do nothing
+	case *driver.Error:
+		if isInvalidArgument(err) {
```

Comment:
I moved such checks to `tigrisdb`, so who merges the second will need to move this one too :P